package/com.unity.formats.usd/Runtime/Scripts/Timeline/UsdRecorderBehaviour.cs

```diff
@@ -129,7 +129,7 @@ public class UsdRecorderBehaviour : PlayableBehaviour {
         Debug.LogError("Process: context.scene is null");
       }
 
-      Clip.UsdScene.Time = currentTime;
+      Clip.UsdScene.Time = currentTime * 60; // conversion to keyframes to work around QuickLook bug
```

Comment:
If you want to add this comment, I would put it near the kExportFrameRate constant.

Also, nit pick: we typically don't put comments on the same line as the code.